### DISCUSSION TITLE
Stripe Webhooks Failing

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#281] [Help] [answered]
Hello,
I'm using the Stripe integration to handle payments on my store. When configuring the integration on Swell, a webhook was configured on stripe. I've now checked the logs, and all calls to `https://core.commercelayer.io/webhook_callbacks/stripe_gateways/<code>` fail with a 404. Any ideas?

**Comment by sags95** [comment, marked as answer]
Hey @igoraugustynski, those webhooks like they belong to commercelayer.io. Is it possible you used the same Stripe account when configuring another platform? When you connect Stripe as a gateway we don't use any webhooks on Stripe.
